Providers/Modules/Plugins/Common/plugin/out_oms_diag.rb

```diff
@@ -0,0 +1,111 @@
+module Fluent
+
+  class OutputOMSDiag < BufferedOutput
+
+    Plugin.register_output('out_oms_diag', self)
```

Comment:
Never mind, I take that back. I think your changes should stay part of nxOMSPlugin, because otherwise we'll either have two copies of oms_configuration.rb or we'll remove oms_configuration.rb from nxOMSPlugin and that is risky.